src/BolWallet/MauiProgram.cs

```diff
@@ -12,31 +12,33 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Options;
 using MudBlazor.Services;
+using Newtonsoft.Json;
```

Comment:
Can this be replaced with System.Text.Json? @moscnik

Reply:
Done